tests/test_operations.py

```diff
@@ -87,21 +90,21 @@ def test_hausdorff_distance(self):
 
     def test_interpolate(self):
         # successful interpolation
-        test_line = LineString(((1,1),(1,2)))
-        known_point = Point(1,1.5)
-        interpolated_point = test_line.interpolate(.5, normalized=True)
+        test_line = LineString([(1, 1), (1, 2)])
```

Comment:
Only intentional change in this file; everything else is autoformatting (or fix on line 33 to keep as one line since autoformatting did bad things there otherwise)